front-end/src/App.js

```diff
@@ -16,7 +17,7 @@ import SubmitTable from "./components/Tables/SubmitTable";
 import LocationViewMap from "./components/Map/LocationViewMap";
 import ViewTable from "./components/Tables/ViewTable";
 
-import Container from '@mui/material/Container';
+import Container from "@mui/material/Container";
```

Comment:
Yeah, prettier defaults to only using double quotes. We should probably create a story to automate using prettier and add configuration that makes it format the way we want. If you prefer single quotes, it can be configured to format that way as well.

I'll make a story and we can discuss the details!

Reply:
Sounds good!

Reply:
Created #81 for this.